modules/configure-ocp-oauth-identity-provider.adoc

```diff
@@ -22,5 +23,13 @@ To integrate the built-in {ocp} OAuth server as an identity provider for {produc
 ====
 For security, Red Hat recommends setting the *Minimum access role* to *None* while you complete setup. Later, you can return to the *Access Control* page to set up more tailored access rules based on user metadata from your identity provider.
 ====
-. To add access rules for users and groups accessing {product-title-short}, use the *Rules* section. For example, to give the *Admin* role to a user called `administrator`, select *User* as *Key*, *administrator* as *Value*, and *Admin* as the *Role*. Use *Add new rule* to add more rules.
+. To add access rules for users and groups accessing {product-title-short}, use the *Rules* section. For example, to give the *Admin* role to a user called `administrator`, select *Name* as *Key*, *administrator* as *Value*, and *Admin* as the *Role*. Use *Add new rule* to add more rules.
 . Click *Save*.
+
+.Example values
+Given an link:https://docs.openshift.com/container-platform/4.9/authentication/understanding-identity-provider.html[HTPasswd Identity Provider] with a user with name "UserA" and group "GroupA",
+the following fields will be available to create access rules:
+
+* "Name"="UserA"
+* "Group"="GroupA"
+* "UserID"="<UUID>"
```

Comment:
. To add access rules for users and groups accessing {product-title-short}, use the *Rules* section. For example:
.. To give the *Admin* role to a user called `administrator`, you can use the following key-value pairs to create access rules:
+
[cols="1,1"]
|===
|Key |Value

|Name
|administrator

|Role
|Admin
|===
.. If you are using the link:https://docs.openshift.com/container-platform/4.9/authentication/understanding-identity-provider.html[HTPasswd Identity Provider] with the username `UserA` that is part of the group `GroupA`, you can use the following key-value pairs to create access rules:
+
[cols="1,1"]
|===
|Key |Value

|Name
|UserA

|Group
|GroupA

|UserID
|<UUID>
|===
. Click *Save*.